frontend/src/pages/destructionlist/detail/DestructionListDetail.action.ts

```diff
@@ -193,7 +193,7 @@ export async function destructionListUpdateZakenAction({
     throw e;
   }
   await clearZaakSelection(storageKey);
-  await cacheDelete("listZaaktypeChoices", true); // Remove possibly outdated zaaktype cache.
+  await cacheDelete("list", true); // Remove possibly outdated cached value of list API's for choices.
```

Comment:
I think that this needs to happen also for the actions "destruction list create" and "destruction list delete", since both of these will affect which choices are available in the create destruction list page.